INSTALL.txt

```diff
@@ -138,18 +160,18 @@ together toolchains for that option. The toolchains are available at
 https://bitbucket.org/carlkl/mingw-w64-for-python/downloads. The site.cfg
 should be configured like so:
```

Comment:
I guess there is an argument for making it valid ReST and then `.. include`'ing it into the docs proper, but eh.